clhep.spec

```diff
@@ -1,6 +1,6 @@
-### RPM external clhep 2.2.0.4
+### RPM external clhep 2.3.1.1
 
-%define tag 0e355c07fbee0b1c680073cc8d77e530a7edc6a9
+%define tag 91f5812f9ade2d61386bd34a2006160ccc0c2909
```

Comment:
@civanch Could you please change this tag to pick up the one corresponding to @davidlt submission? (87505534a473b0195037fae21575cbdbf53bfbbb)